inst/sympref.m

```diff
@@ -420,6 +420,13 @@
 %! delete('tmp_python_cmd.py')
 
 %!test
+%! warning ('off', 'OctSymPy:sympref:invalidarg', 'local');
```

Comment:
Looks like we the pattern `s = warning ('off', ...);` then `warning (s)` 

IIRC, that's because my hack to run the tests on Matlab doesn't put them in their own scope.

Maybe I'll fix that post-merge which has the nice side-effect of documenting this in the git log.